bin/check-influxdb-metrics.rb

```diff
@@ -108,13 +125,20 @@ def yesterday_query_for_a_period
   end
 
   def query_for_a_period(start_period, end_period)
-    query = base_query + ' WHERE time > now() - ' + end_period.to_s + 'm AND time < now() - ' + start_period.to_s + 'm'
+    query = base_query_with_metricname + ' WHERE time > now() - ' + end_period.to_s + 'm AND time < now() - ' + start_period.to_s + 'm'
     query + filter_by_environment_when_needed
   end
 
   def encode_parameters(parameters)
     encodedparams = Addressable::URI.escape(parameters)
-    "#{config[:db]}&q=" + encodedparams
+
+    encode_for_regex = if @is_using_regex
+                         encodedparams.gsub! '+', '%2B'
```

Comment:
Sure! Will do. It will come in the next change including documentation and the new tag. Thanks @andyroyle !